demos/serverless/src/handlers.js

```diff
@@ -135,7 +135,7 @@ exports.logs = async (event, context) => {
   try {
     await cloudWatchClient.putLogEvents(putLogEventsInput).promise();
   } catch (error) {
-    console.error(`Failed to put CloudWatch log events with error ${error.message} and params ${JSON.stringify(putLogEventsInput)}`);
+    console.warn(`Failed to put CloudWatch log events with error ${error.message} and params ${JSON.stringify(putLogEventsInput)}`);
```

Comment:
We need to log error for everything except the DataAlreadyAcceptedException since we decided to skip that one.

Reply:
Why are we doing this anyways?

Reply:
We decide to not handle DataAlreadyAcceptedException so just want to catch and skip that error.